src/transaction_signature_options/mod.rs

```diff
@@ -21,7 +22,7 @@ pub const META_TRANSACTION_VALID_FOR_DEFAULT: u64 = 1000;
 pub enum SignWith {
     #[cfg(target_os = "macos")]
     #[strum_discriminants(strum(
-        message = "sign-with-macos-keychain         - Sign the transaction with a key saved in macOS keychain"
+        message = "sign-with-macos-keychain         - Sign the transaction with a key saved in macOS keychain (Online only!)"
```

Comment:
We agreed that we don't need to confuse users with "online mode" as it is rare case when someone would want to use near-cli-rs in offline mode, so revert this change (remove the "(Online only!)" note)

Reply:
Done